core/common/src/main/java/alluxio/grpc/GrpcManagedChannelPool.java

```diff
@@ -227,7 +258,7 @@ private ManagedChannel get(){
    */
   public static class ChannelKey {
     private SocketAddress mAddress;
-    private boolean mPlain = false;
+    private boolean mPlain = true;
```

Comment:
Why is this changed?

Reply:
I thought plain should be true as default in OS code base as it's the case.